### PR TITLE
Send Email to User While register

### PR DESCRIPTION
**Pull request by SaniyaKumawat123**
Send Email to User While register

### CODE REVIEW

File: src/main/java/com/notification/controller/NotificationController.java [hunk 1]
```diff
@@ -70,4 +70,11 @@ public ResponseEntity<ResponseModel<InvitationStatusResponse>> updateStatus(
         log.info("Endpoint: /put update status ends with update status: {}", response.getStatus());
         return ResponseEntity.ok(response);
     }
+
+    @PostMapping(value = "/send-registration-mail", produces = MediaType.APPLICATION_JSON_VALUE)
+    public void sendConfirmationMail(@RequestBody InviteRequest inviteRequest) {
```
Comment (Jatin-Ongraph) [inline src/main/java/com/notification/controller/NotificationController.java:75]: Remove inviteRequest and use PathVariable as sentOnEmail

File: src/main/java/com/notification/service/impl/NotificationServiceImpl.java [hunk 2]
```diff
@@ -127,6 +129,21 @@ public ResponseModel<InvitationStatusResponse> updateInviteeStatus(ObjectId invi
 		return response;
 	}
 
+	@Override
+	public void sendRegistrationMail(InviteRequest inviteRequest) {
+
+		try {
+			String confirmationLink = String.format(CONFIRMATION_LINK, authBaseUrl, inviteRequest.getSentToEmail());
+			Context thymeleafContext = new Context();
+			thymeleafContext.setVariable("confirmationLink", confirmationLink);
```
Comment (Jatin-Ongraph) [inline src/main/java/com/notification/service/impl/NotificationServiceImpl.java:138]: Replace string literals with constant final variables.

File: src/main/java/com/notification/controller/NotificationController.java (hunk 1 above)
Comment (SaniyaKumawat123) [reply src/main/java/com/notification/controller/NotificationController.java:75]: I am using two variables here, which are user email and user name, which is why I am using the existing InviteRequest.

File: src/main/java/com/notification/service/impl/NotificationServiceImpl.java (hunk 2 above)
Comment (SaniyaKumawat123) [reply src/main/java/com/notification/service/impl/NotificationServiceImpl.java]: Updated

File: src/main/java/com/notification/controller/NotificationController.java (hunk 1 above)
Comment (Jatin-Ongraph) [reply src/main/java/com/notification/controller/NotificationController.java:80]: Ok

**Review by Jatin-Ongraph — APPROVED** [review]

**Pull request merged**